src/studio/studio.c

```diff
@@ -2313,19 +2313,13 @@ static void blitCursor(Studio* studio)
         if(tic->ram->vram.vars.cursor.system)
         {
             bank = &getConfig(studio)->cart->bank0;
-            hot = (tic_point[])
```

Comment:
Why are you deleting hardcoded "hot" points of the cursor here?

Reply:
Didn't know why the hot code was responsible for duplicating the mouse sprites. Worked great without it and made it simpler. So I left it that way.

Reply:
We use hot point in the code editor for example for IBeam cursor

Reply:
The IBeam cursor works even without the hot point.

Reply:
it works without hotpoint, but the click point should be the center of the I Beam cursor

Reply:
I see, I'll look into that

Reply:
Found the proper fix, ready to be merged